math/minuit2/inc/Minuit2/Minuit2Minimizer.h

```diff
@@ -239,16 +239,21 @@ class Minuit2Minimizer : public ROOT::Math::Minimizer {
       get the minos error for parameter i, return false if Minos failed
       A minimizaiton must be performed befre, return false if no minimization has been done
       In case of Minos failed the status error is updated as following
-      status += 10 * minosStatus where the minos status is:
-       status = 1    : maximum number of function calls exceeded when running for lower error
-       status = 2    : maximum number of function calls exceeded when running for upper error
-       status = 3    : new minimum found when running for lower error
-       status = 4    : new minimum found when running for upper error
-       status = 5    : any other failure
-
+      status += 10 * minosStatus. 
+      The Minos status of last Minos run can also be retrieved bby calling MinosStatus()
    */
    virtual bool GetMinosError(unsigned int i, double & errLow, double & errUp, int = 0);
 
+   /** 
+      MINOS status code of last Minos run
+       status & 1 > 0  : invalid lower error
+       status & 2 > 0  : invalid upper error 
+       status & 4 > 0  : invalid because maximum number of function calls exceeded 
+       status & 8 > 0  : a new minimum has been found
+       status & 16 > 0 : error is truuncated because parameter is at lower/upper limit
```

Comment:
```suggestion
       - `status & 1 > 0` : invalid lower error
       - `status & 2 > 0` : invalid upper error 
       - `status & 4 > 0` : invalid because maximum number of function calls exceeded 
       - `status & 8 > 0` : a new minimum has been found
       - `status & 16 > 0` : error is truncated because parameter is at lower/upper limit
```